cmd/osquery-perf/ubuntu_22.04.tmpl

```diff
@@ -23,58 +25,57 @@
             "platform_mask": "21",
             "start_time": "1580931224",
             "uuid": "{{ .UUID }}",
-            "version": "4.6.0",
-            "watcher": "12946"
+            "version": "5.5.1",
+            "watcher": "-1"
         },
         "platform_info": {
-            "address": "0xff990000",
-            "date": "12/16/2019 ",
-            "extra": "MBP114; 196.0.0.0.0; root@xapp160; Mon Dec 16 15:55:18 PST 2019; 196 (B&I); F000_B00; Official Build, Release; Apple LLVM version 5.0 (clang-500.0.68) (based on LLVM 3.3svn)",
-            "revision": "196 (B&I)",
-            "size": "8388608",
-            "vendor": "Apple Inc. ",
-            "version": "196.0.0.0.0 ",
-            "volume_size": "1507328"
         },
         "system_info": {
             "computer_name": "{{ .CachedString "hostname" }}",
-            "cpu_brand": "Intel(R) Core(TM) i7-4770HQ CPU @ 2.20GHz\u0000\u0000\u0000\u0000\u0000\u0000\u0000",
+            "cpu_brand": "Intel(R) Core(TM) i7-4770HQ CPU @ 2.20GHz",
             "cpu_logical_cores": "8",
             "cpu_physical_cores": "4",
             "cpu_subtype": "Intel x86-64h Haswell",
             "cpu_type": "x86_64h",
-            "hardware_model": "MacBookPro11,4",
-            "hardware_serial": "D02R835DG8WK",
-            "hardware_vendor": "Apple Inc.",
-            "hardware_version": "1.0",
+            "hardware_model": "",
+            "hardware_serial": "",
+            "hardware_vendor": "",
+            "hardware_version": "",
             "hostname": "{{ .CachedString "hostname" }}",
             "local_hostname": "{{ .CachedString "hostname" }}",
             "physical_memory": "17179869184",
             "uuid": "{{ .UUID }}"
         }
     },
-    "host_identifier": "{{ .CachedString "hostname" }}",
+    "host_identifier": "{{ .UUID }}",
```

Comment:
I'm impressed that you catch this one (bot here and in the Windows template), it's super important for Orbit enrollment. Sorry for not updating it.